rustmodels/src/medrecord/querying/attributes.rs

```diff
@@ -0,0 +1,565 @@
+use super::values::PyMultipleValuesOperand;
+use crate::medrecord::{attribute::PyMedRecordAttribute, errors::PyMedRecordError};
```

Comment:
error or can it be multiple?

Reply:
the module is called `errors`. It's called `errors` because `rustmodels` is mostly a 1 to 1 representation of the modules in `medmodels-core` which also has an `errors` module and there are multiple error types in that module